bcda/utils/etl_test.go

```diff
@@ -0,0 +1,59 @@
+package utils
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+type EtlTestSuite struct {
```

Comment:
```suggestion
type ETLTestSuite struct {
```
https://github.com/golang/go/wiki/CodeReviewComments#initialisms